src/Data/Set.purs

```diff
@@ -73,3 +76,16 @@ unions = foldl union empty
 
 difference :: forall a. (P.Ord a) => Set a -> Set a -> Set a
 difference s1 s2 = foldl (P.flip delete) s1 (toList s2)
+
+subset :: forall a. (P.Ord a) => Set a -> Set a -> Boolean
+subset s1 s2 = isEmpty P.$ s1 `difference` s2
+
+properSubset :: forall a. (P.Ord a) => Set a -> Set a -> Boolean
+properSubset s1 s2 = subset s1 s2 P.&& (P.not P.<<< isEmpty P.$ s2 `difference` s1)
```

Comment:
This is a little hard to read due to all the `P.`s. Maybe we should using `import Prelude hiding (empty)` instead, or just use parens and a lambda here.